aiohttp/client.py

```diff
@@ -542,8 +544,13 @@ def request(self,
             # check response status
             if raise_for_status is None:
                 raise_for_status = self._raise_for_status
-            if raise_for_status:
+
+            if raise_for_status is True:
```

Comment:
There is a subtle regression here.
Before a user was allowed to pass `1` / `0` or even an object with `__bool__()` dunder method implemented.
We discourage this by providing `bool` type hint but still support.
Now the check works by explicit `is True` comparison.
I rather suggest reorder checks:
```
if raise_for_status is None:
    pass
elif callable(raise_for_status):
    await raise_for_status(resp)
elif raise_for_status:  # a shortcut for `bool(raise_for_status)`
    resp.raise_for_status()
```
What do you think?

Reply:
done